build/config/compiler/BUILD.gn

```diff
@@ -66,20 +66,6 @@ config("compiler") {
   ldflags = []
   defines = []
 
-  # We want to force a recompile and relink of the world whenever our toolchain
-  # changes since artifacts from an older version of the toolchain may or may
-  # not be compatible with newer ones. To achieve this, we insert a synthetic
-  # define into the compile line. We only do this for toolchains that we use
-  # from buildtools.
-  toolchain_version =
-      exec_script("//build/util/lastchange.py",
```

Comment:
Can we drop this script as well then?

Reply:
It seems to be referenced in another rule (that we don't use anyway). Removing it was making me go down a yak shave of manually finding if each file was being used.